next/tailwind.config.js

```diff
@@ -270,6 +300,9 @@ module.exports = {
         61: '244px',
         87: '350px',
       },
+      gap: {
+        3: '13px'
```

Comment:
still not convinced about this one but let's deal with it in the global spacing rewrite (will create a JIRA task today)